src/Components/ExplorePage/partials/ProjectsRow.tsx

```diff
@@ -34,7 +34,7 @@ export default function ProjectsRow({ title, categoryId, projects }: Props) {
     document.addEventListener('mousemove', () => drag.current = true);
 
```

Comment:
Use the newly pushed Commit to the original repo where the real id is passed
and almost all objects ids now are number instead of string